keymanapp/keyman
# chore(common): Check in crowdin for Ukrainian

Adds crowdin strings for Ukrainian for locale uk-UA

## User Testing
This adds the crowdin strings for Ukrainian locale uk-UA

TODO for @sgschantz 

## User Testing
Load the PR build for each platform and set the UI / Locale to "Українська (Ukrainian)"

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> Українська (Ukrainian)
3. Return to the Keyman app 
4. Verify the UI is in Ukrainian (prompt, Settings strings, etc.).



* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Ukrainian
3. Return to the Keyman app
4. Verify the UI is in  Ukrainian

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=uk-UA km-config
2. Verify Keyman UI is in Ukrainian

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the locale to Ukrainian
3. Verify the Keyman UI is in Ukrainian

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to Українська (Ukrainian)
3. Verify the Keyman UI is in Ukrainian
